pkg/gotohelm/helmette/sprig.go

```diff
@@ -14,6 +14,15 @@ import (
 	"golang.org/x/exp/maps"
 )
 
+// +gotohelm:builtin=first
+func First[T any](x []T) any {
+	if len(x) == 0 {
+		return nil
```

Comment:
~Does this work? `nil` would be an invalid return if `T` is any primitive (string, int). I would expect the go compiler to yell at you.~ OH, you're returning `any`.

Depending on what sprig does internally, it might be preferable to make this `func[T any]([]T) T`? The syntax is a bit weird to make it work though
```go
if len(x) == 0 {
  var zero T
  return zero
}


```

Reply:
That was my initial implementation, but then go code gives different output. Please see

https://github.com/Masterminds/sprig/blob/581758eb7d96ae4d113649668fa96acc74d46e7f/list.go#L163

To match implementation I would leave it as is, WDYT?

Reply:
Ah. Hm. That's a tricky one. Mind adding that link as a comment for future reference? It might be painful to deal with `any` but we can also make improvements in the future if need be.

Reply:
Yes, I will add comment.